retrofit/src/test/java/retrofit/RetrofitTest.java

```diff
@@ -115,10 +115,29 @@
     }
   }
 
+  @Test public void validateEagerlyDisabledByDefault() {
+    Retrofit retrofit = new Retrofit.Builder()
+        .baseUrl(server.url("/"))
+        .build();
+
+    // Should not throw exception about incorrect configuration of the VoidService
+    retrofit.create(VoidService.class);
+  }
+
+  @Test public void validateEagerlyDisabledByUser() {
+    Retrofit retrofit = new Retrofit.Builder()
+        .baseUrl(server.url("/"))
+        .validateEagerly(false)
+        .build();
+
+    // Should not throw exception about incorrect configuration of the VoidService
```

Comment:
Missing a period here and above. [Comments should be sentences][1].

 [1]: nedbatchelder.com/blog/201401/comments_should_be_sentences.html